docs/ldml/tr35-dates.md

```diff
@@ -2365,6 +2365,314 @@ The meaning of symbol fields should be easy to determine; the problem is determi
+
+| Field Set                         | Example                   |
+|-----------------------------------|---------------------------|
+| { Day }                           | The 1st                   |
+| { Weekday }                       | Saturday                  |
+| { Day, Weekday }                  | Saturday the 1st          |
+| { Month, Day }                    | January 1                 |
+| { Month, Day, Weekday }           | Saturday, January 1       |
+| { Year, Month, Day }              | January 1, 2000           |
+| { Year, Month, Day, Weekday }     | Saturday, January 1, 2000 |
+
+Note: Month and Year are not valid date field sets on their own because they do not refer to a specific day. Instead, they are considered calendar period field sets.
+
+Note: This table may be extended in the future to include additional fields, such as week and quarter.
+
+##### <a name="Semantic_Calendar_Period_Field_Sets" href="#Semantic_Calendar_Period_Field_Sets">Calendar Period Field Sets</a>
+
+A _calendar period field set_ refers to a span of time in a calendar system, _above_ the order of a day.
+
+The fields that are permissible in date field sets are also the ones permissible in calendar period field sets.
+
+The valid calendar period field sets are in the following table:
+
+| Field Set           | Example      |
+|---------------------|--------------|
+| { Month }           | January      |
+| { Year }            | 2000         |
+| { Year, Month }     | January 2000 |
+
+Note: This table may be extended in the future to include additional fields, such as week, quarter, or standalone era.
+
+Note: A _calendar period_ is distinct from a _date_ because it cannot be paired with time to form a composite field set.
+
+##### <a name="Semantic_Time_Field_Sets" href="#Semantic_Time_Field_Sets">Time Field Sets</a>
+
+A _time field set_ refers to a particular time of day. Low-order fields, such as the minute or second, could be omitted.
+
+The fields that may be included in a time field set are:
+
+1. **Hour:** The hour, possibly with a day period, depending on factors such as the length, locale, and hour cycle locale keyword.
+2. **Minute:** The minute within an hour.
+3. **Second:** The second within a minute, a decimal number that may include fractional digits. See the [Fractional Second Digits](#Semantic_Skeleton_Fractional_Second_Digits) option.
+
+The valid time field sets are in the following table:
+
+| Field Set                | Example (en-US) | Example (en-GB) |
+|--------------------------|-----------------|-----------------|
+| { Hour }                 | 4 pm            | 16h             |
+| { Hour, Minute }         | 4:03 pm         | 16:03           |
+| { Hour, Minute, Second } | 4:03:51 pm      | 16:03:51        |
+
+Note: Minute and Second are not valid time field sets on their own because they do not refer to a particular time of day. They must be interpreted in the context of an explicit hour.
```

Comment:
Add a note somewhere that formatting **durations**, like 3 minutes 12 seconds (or 3:12) is not handled through the  Skeleton mechanisms.